pages/settings/settings.js

```diff
@@ -150,13 +149,35 @@ for (var contentType in contentTypes) {
 }
 
 /* dark mode setting */
-
-settings.get('darkMode', function (value) {
-  darkModeCheckbox.checked = value
+var darkModeNever = document.getElementById('dark-mode-never')
+var darkModeNight = document.getElementById('dark-mode-night')
+var darkModeAlways = document.getElementById('dark-mode-always')
+
+// -1 off ; 0 - auto ; 1 - on
+var darkMode = -1
+settings.get('darkMode', function(value) {
+  darkMode = value;
 })
 
-darkModeCheckbox.addEventListener('change', function (e) {
-  settings.set('darkMode', this.checked)
+  /* Setting up the RB's */
+    darkModeNever.checked = (darkMode === -1)
+    darkModeNight.checked = (darkMode === 0 || darkMode === undefined || darkMode === false) 
+    darkModeAlways.checked = (darkMode === 1 || darkMode === true)
```

Comment:
`settings.get` is asynchronous when called from the preferences page, so these should go inside the callback function.